psi4/driver/procrouting/proc.py

```diff
@@ -1636,12 +1646,15 @@ def scf_helper(name, post_scf=True, **kwargs):
                  scf_wfn.epsilon_b(), scf_wfn.occupation_a(),
                  scf_wfn.occupation_b(), dovirt)
 
-    # Write out orbitals and basis; Can be disabled, e.g., for findif displacements
-    if kwargs.get('write_orbitals', True):
-        write_filename = scf_wfn.get_scratch_filename(180)
-
-        scf_wfn.to_file(write_filename)
-        extras.register_numpy_file(write_filename)
+    # Write checkpoint file (orbitals and basis); Can be disabled, e.g., for findif displacements
```

Comment:
This reminds me - when we're doing finite difference with orbital reading, we want to read in the undisplaced orbitals as the guess for all displacements. We don't want to read in the orbitals from a previous displacement. Does this PR respect that?

Reply:
No, that will need to be fixed. The `restart_file` orbitals is used for all displacement now.

Previously all displacement calcs got a SAD guess? (Since orbital files are not written)

Reply:
What was _supposed_ to happen and did happen as of #1424 is that if `guess read` was enabled, the undisplaced goemetry would write its orbitals, those would be the only orbitals written, and those would be read in for all future displacements. If the orbital files aren't being written, somebody broke that feature in another PR, probably without thinking about it.

Reply:
Fixed. findif gradient/hessian calls with `restart_file` only use the restart_file for the reference geometry, displacements load the orbitals from the reference geometry's SCF.